About.html

```diff
@@ -0,0 +1,137 @@
+<!DOCTYPE html>
+<html lang="en">
+
+<head>
+    <meta charset="UTF-8">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <link rel="stylesheet" href="styles.css">
+    <script src="https://code.jquery.com/jquery-3.6.0.min.js"></script>  
+    <link href="https://fonts.googleapis.com/css2?family=Lato:wght@400;700;900" rel="stylesheet">
+    <title>Document</title>
+</head>
+
+<body>
+
+    <header>
+        <div id="black-header " class="display_desktop"> 
+            <ul class="menu-black">
+               
+                <li><img src="images/facebook.png"></li>
+                <li><img src="images/twitter.png"></li>
+                <li>Logout</li>
+                <li>Mypage</li>
+                <li>Korean</li>
+            </ul>
+            </div>
+        <nav class="display_desktop navigation">
+            
+            <ul class="desktop_menu">
+                <li> <img src="images/logo.png" class="logo"></li>
+                <li>Program</li>
```

Comment:
- [ ] As mentioned by the previous reviewer, the current page is the About page, and it doesn't make sense to have an anchor tag linking to the same page. You correctly followed their advice and removed the link. However, they also suggested changing the text from "About" to "Home." Because currently, there is no direct way to navigate back to the home page from the About page. 🏠 To improve user experience and navigation, please update your code so that users can easily navigate from the About page (about.html) to the home page (index.html). You can achieve this by adding the appropriate links in the navigation menu by adding a home nav item with proper link. 📌 This will make the website more user-friendly and enhance overall usability. Thank you for your attention to this matter! 🙏

| Your | Professional |
| --- | --- |
| ![image](https://github.com/umarahgul/capstone-1/assets/55840999/105d1a64-d3d8-426a-ad32-77a98ff71921) | ![image](https://github.com/umarahgul/capstone-1/assets/55840999/08afe7d8-0626-4c96-8091-b1cb3d6eb8cf) |